attestation-agent/attester/Cargo.toml

```diff
@@ -19,7 +19,7 @@ sev = { version = "1.2.0", default-features = false, features = ["snp"], optiona
 strum.workspace = true
 tdx-attest-rs = { git = "https://github.com/intel/SGXDataCenterAttestationPrimitives", tag = "DCAP_1.16", optional = true }
 # TODO: change it to "0.1", once released.
-csv-rs = { git = "https://gitee.com/anolis/csv-rs", rev = "bcf3bcc", optional = true }
+csv-rs = { git = "https://gitee.com/anolis/csv-rs", rev = "05fbacd", optional = true }
```

Comment:
verified, this fix the issue indeed, 

```
cargo test --features openssl,rust-crypto,offline_fs_kbc -p attestation-agent -p attester -p coco_keyprovider -p kbc -p kbs_protocol -p attestation_agent -p crypto -p resource_uri -p sev@0.1.0
```

passed on my desktop!

thanks @BaoshunFang !


cc @Xynnn007